src/main/java/com/opentable/db/postgres/embedded/ProcessOutputLogger.java

```diff
@@ -63,7 +63,7 @@ public void run() {
 
     static void logOutput(final Logger logger, final Process process) {
         final Thread t = new Thread(new ProcessOutputLogger(logger, process));
-        t.setName("output redirector for " + process);
+        t.setName("pg-" + process.pid());
```

Comment:
@stevenschlansker I merged this but I'm reverting, because process.pid() is Java9+ only. For now I'd prefer to keep a utility like this Java 8 compatible.

Reply:
@stevenschlansker @mikebell90 What do you guys think about a backwards-compatible implementation where we detect (via reflection) whether the object has a `pid` method or not and act accordingly?

Reply:
@pennello @stevenschlansker Good to hear from you both! Anyway sure that would be ok. 

I do think it's reasonable to keep a tool like this Java 8 compatible by default for a good period longer.